snyk/utils.py

```diff
@@ -19,3 +20,33 @@ def flat_map(fn, *args):
 
 def format_package(pkg):
     return "{name}@{version}".format(name=pkg.name, version=pkg.version or "*")
+
+
+def cleanup_path(path: str, logger: logging.Logger) -> str:
+    """
+    Strings '/' from the start and end of strings if present to ensure that a '//' doesn't
+    occur in an API request due to copy/paste error
+    """
+
+    if path[0] == "/":
+        logger.warn(f"WARNING: removing unneccessary leading / from {path}")
```

Comment:
Given we're indicating it as a warning, I don't think we should repeat that in the returned string. The logger should already surface the type.